packages/manager/apps/pci-object-storage/src/api/data/storages.ts

```diff
@@ -106,3 +106,159 @@ export const updateStorage = async ({
 
   return data;
 };
+
+export const createSwiftStorage = async ({
+  projectId,
+  archive,
+  containerName,
+  region,
+}: {
+  projectId: string;
+  archive: boolean;
+  containerName: string;
+  region: string;
+}) => {
+  const { data } = await v6.post<TStorage>(
+    `/cloud/project/${projectId}/storage`,
+    {
+      archive,
+      containerName,
+      region,
+    },
+  );
+
+  return data;
+};
+
+export const createS3Storage = async ({
+  projectId,
+  containerName,
+  ownerId,
+  region,
+  encryption,
+  versioning,
+}: {
+  projectId: string;
+  containerName: string;
+  region: string;
+  ownerId: string;
+  encryption: string;
+  versioning: boolean;
+}) => {
+  const { data } = await v6.post<TStorage>(
+    `/cloud/project/${projectId}/region/${region}/storage`,
+    {
+      name: containerName,
+      ownerId,
+      encryption: {
+        sseAlgorithm: encryption,
+      },
+      ...(versioning
+        ? {
+            versioning: {
+              status: versioning ? 'enabled' : 'disabled',
+            },
+          }
+        : {}),
+    },
+  );
+
+  return data;
+};
+
+export type TStorageAccess = {
+  endpoints: { region: string; url: string }[];
+  token: string;
+};
+
+export const getStorageAccess = async ({
```

Comment:
it's duplicated there is the same call in object.ts 

```
export const getAccessToken = async (projectId: string): Promise<TAccess> => {
  const { data } = await v6.post<TAccess>(
    `/cloud/project/${projectId}/storage/access`,
  );
  return data;
};
```

Reply:
good catch i refactored this part